spec/cspec/c/kernel.mk

```diff
@@ -127,6 +119,59 @@ ${KERNEL_CONFIG_ROOT}/.cmake_done: ${KERNEL_DEPS} gen-config-thy.py
 	cmake -C ${CONFIG} \
 		-DCMAKE_TOOLCHAIN_FILE=${CSPEC_DIR}/c/no-compiler.cmake \
 		${KERNEL_CMAKE_EXTRA_OPTIONS} \
-		-G Ninja ${SOURCE_ROOT}
+		-G Ninja ${KERNEL_SOURCE_ROOT}
 	cd ${KERNEL_CONFIG_ROOT} && ninja gen_config/kernel/gen_config.json
 	touch ${KERNEL_CONFIG_ROOT}/.cmake_done
+
+# Various targets useful for binary verification.
+${KERNEL_BUILD_ROOT}/kernel.elf: ${KERNEL_BUILD_ROOT}/kernel_all.c_pp
+	cd ${KERNEL_BUILD_ROOT} && ninja kernel.elf
+
+${KERNEL_BUILD_ROOT}/kernel.elf.rodata: ${KERNEL_BUILD_ROOT}/kernel.elf
+	${OBJDUMP} -z -D $^ > $@
+
+${KERNEL_BUILD_ROOT}/kernel.elf.txt: ${KERNEL_BUILD_ROOT}/kernel.elf
+	${OBJDUMP} -dz $^ > $@
+
+${KERNEL_BUILD_ROOT}/kernel.elf.symtab: ${KERNEL_BUILD_ROOT}/kernel.elf
+	${OBJDUMP} -t $^ > $@
+
+# Override STANDALONE_C_PARSER_EXE to use a pre-built standalone C parser.
+ifndef STANDALONE_C_PARSER_EXE
+  STANDALONE_C_PARSER_DIR := ${L4V_REPO_PATH}/tools/c-parser/standalone-parser
+  STANDALONE_C_PARSER_EXE := ${STANDALONE_C_PARSER_DIR}/${L4V_ARCH}/c-parser
+endif
```

Comment:
This doesn't seem to override as the comment says, but instead provide a default. Am I reading that correctly?

Reply:
Or is the comment trying to say that the user can override this variable to use a different standalone parser?

Reply:
Oh, I didn't consider that reading of the comment. In my head, it said "If you want to use a pre-built standalone C parser, then set STANDADLONE_C_PARSER_EXE so this default isn't used."

I will try to improve that.

Reply:
> Or is the comment trying to say that the user can override this variable to use a different standalone parser?

Our replies crossed paths, but yes, that is how it read in my head. :-)

I do acknowledge the ambiguity, which I will try to fix.